integreat_cms/cms/templates/offertemplates/offertemplate_form.html

```diff
@@ -71,6 +71,15 @@ <h3 class="heading font-bold text-black">
                         </div>
                         {% render_field form.thumbnail %}
                     </div>
+                    <div>
+                        <label for="{{ form.supported_by_app_in_content.id_for_label }}">
```

Comment:
```suggestion
                        <label for="{{ form.supported_by_app_in_content.id_for_label }}" class="flex items-center">
```
I found that checkbox and label weren't aligned (s.screenshot). Items-center should do :)
![grafik](https://github.com/digitalfabrik/integreat-cms/assets/72705147/d5ba6a49-8cc9-4684-b864-9bbec1b4778f)